cli/module_generate/_templates/python/src/models/tmpl-resource.py

```diff
@@ -9,6 +10,9 @@
 
 class {{ .ModelPascal  }}({{ .ResourceSubtypePascal }}, EasyResource):
     MODEL: ClassVar[Model] = "{{ .ModelTriple }}"
+    # To enable debug-level logging, either run viam-server with the --debug option,
+    # or configure your resource/machine to display debug logs.
+    logger = logging.getLogger(__name__)
```

Comment:
This is the old pattern for creating (relatively) global loggers (this logger will is attached to the file so will always claim to have been made by a `my_resource` without specifying which). I think we can (and should) get rid of this invocation as it's a bit confusing; the logger on the resource object is created in the `new` method automatically.